packages/react-ui-core/src/DropDown/DropDown.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import DefaultAnchor from './DefaultAnchor'
+
+@themed(/^DropDown/)
+
+export default class DropDown extends Component {
```

Comment:
What do you think about calling this `Dropdown` instead?  This casing seems to be the norm if you do a search on the web.  

https://v4-alpha.getbootstrap.com/components/dropdowns/
https://semantic-ui.com/modules/dropdown.html
http://materializecss.com/dropdown.html

Reply:
Yea, that works. We should probably follow that convention.